pom.xml

```diff
@@ -181,54 +181,51 @@
         <version>${tycho.version}</version>
         <extensions>true</extensions>
       </plugin>
-        <plugin>
-          <groupId>org.eclipse.tycho</groupId>
-          <artifactId>target-platform-configuration</artifactId>
-          <version>${tycho.version}</version>
-          <configuration>
-            <dependency-resolution>
-              <extraRequirements>
-                <!-- Extra Target Bundles Here -->
-              </extraRequirements>
-              <!-- Default behavior is to resolve ALL dependencies. -->
-              <!--<optionalDependencies>ignore</optionalDependencies>-->
-            </dependency-resolution>
+      <plugin>
+        <groupId>org.eclipse.tycho</groupId>
+        <artifactId>target-platform-configuration</artifactId>
+        <version>${tycho.version}</version>
+        <configuration>
+          <dependency-resolution>
+            <extraRequirements>
+              <!-- Extra Target Bundles Here -->
+            </extraRequirements>
+            <!-- Default behavior is to resolve ALL dependencies. -->
+            <!--<optionalDependencies>ignore</optionalDependencies> -->
+          </dependency-resolution>
 
-            <filters>
-              <!-- This is how features/plugins can be ignored. -->
-              <!--<filter>
-                <type>eclipse-plugin</type>
-                <id>artifactId</id>
-                <removeAll />
-              </filter>-->
-            </filters>
+          <filters>
+            <!-- This is how features/plugins can be ignored. -->
+            <!--<filter> <type>eclipse-plugin</type> <id>artifactId</id> 
+              <removeAll /> </filter> -->
+          </filters>
 
-            <environments>
-              <environment>
-                <os>linux</os>
-                <ws>gtk</ws>
-                <arch>x86</arch>
-              </environment>
-              <environment>
-                <os>linux</os>
-                <ws>gtk</ws>
-                <arch>x86_64</arch>
-              </environment>
-              <environment>
-                <os>win32</os>
-                <ws>win32</ws>
-                <arch>x86</arch>
-              </environment>
-              <environment>
-                <os>win32</os>
-                <ws>win32</ws>
-                <arch>x86_64</arch>
-              </environment>
-              <environment>
-                <os>macosx</os>
-                <ws>cocoa</ws>
-                <arch>x86</arch>
-              </environment>
+          <environments>
+            <environment>
+              <os>linux</os>
+              <ws>gtk</ws>
+              <arch>x86</arch>
```

Comment:
Same at SNS: For Linux, we need both 64 and 32 bit.
Windows and Mac are only 64.